check-environment.js

```diff
@@ -1,22 +1,24 @@
-
-const auth = require('@adobe/jwt-auth')
-const fs = require('fs')
 const fetch = require('node-fetch')
 
 require('dotenv').config()
 
 async function getAccessToken () {
-  const config = {
-    clientId: process.env.CLIENT_ID,
-    technicalAccountId: process.env.TECHNICAL_ACCOUNT_ID,
-    orgId: process.env.ORGANIZATION_ID,
-    clientSecret: process.env.CLIENT_SECRET,
-    metaScopes: [ 'ent_cloudmgr_sdk' ]
+  const form = new FormData();
```

Comment:
are we gonna remove JWT access token? I think we should keep both type of support for now.

Reply:
The description mentions to update it to OAuth instead of JWT, so did the changes accordingly. But yes will cross check this with Kritika.

Reply:
well we had recent conversations in the other PR to support both

Reply:
maybe you can just put the older function in a commented format so that its available to use

Reply:
added jwt token access in commented format and also updated the jira description.